packages/web-shared/src/api/workflow-server-actions.ts

```diff
@@ -477,10 +477,12 @@ export async function cancelRun(
 ): Promise<ServerActionResult<void>> {
   try {
     const world = getWorldFromEnv(worldEnv);
-    await world.runs.cancel(runId);
+    await world.events.create(runId, { eventType: 'run_cancelled' });
     return createResponse(undefined);
   } catch (error) {
-    return createServerActionError<void>(error, 'world.runs.cancel', { runId });
+    return createServerActionError<void>(error, 'world.events.create', {
+      runId,
+    });
```

Comment:
@TooTallNate : this change will cause o11y to break when trying to cancel runs against an old version

Reply:
Will it? It'll hit the v2 endpoint since this version will be shipped with world-vercel v2, and then properly update the run materialization, so I'd assume this is safe to do?
Unless you're thinking the run will find the unsupported event in the event log, and then throw, which IMO seems fine too, since we're cancelling anyway and don't want the run to keep running